api/rosetta/errors.go

```diff
@@ -193,3 +204,69 @@ func invalidIntent(fail failure.InvalidIntent) Error {
 		fail.Description,
 	)
 }
+
+func invalidAuthorizers(fail failure.InvalidAuthorizers) Error {
+	return convertError(
+		configuration.ErrorInvalidAuthorizers,
+		fail.Description,
+		withDetail("have_authorizers", fail.Have),
+		withDetail("want_authorizers", fail.Want),
+	)
+}
+
+func invalidPayer(fail failure.InvalidPayer) Error {
+	return convertError(
+		configuration.ErrorInvalidPayer,
+		fail.Description,
+		withAddress("have_payer", fail.Have),
+		withAddress("want_payer", fail.Want),
+	)
+}
+
+func invalidProposer(fail failure.InvalidProposer) Error {
+	return convertError(
+		configuration.ErrorInvalidProposer,
+		fail.Description,
+		withAddress("have_proposer", fail.Have),
+		withAddress("want_proposer", fail.Want),
+	)
+}
+
+func invalidScript(fail failure.InvalidScript) Error {
+	return convertError(
+		configuration.ErrorInvalidScript,
+		fail.Description,
```

Comment:
Maybe we would want to include the script here? WDYT? Too much?

Reply:
Yeah, I think it's useful. I've added it.

However, I did not add it in the `InvalidScript` `Error()` method. The transaction text is usually pretty long and I think it would make the error unreadable. Do you think it should be added? It's the only `Error()` method that deviates from the rule I think.